src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -587,6 +600,7 @@ export class DynamicFacet extends Component implements IAutoLayoutAdjustableInsi
     this.position = index + 1;
 
     this.onQueryResponse(response);
+    this.updateQueryStateModel();
```

Comment:
This will be necessary for autoselection updating the QSM
(will add a test about it)